java/main/src/main/java/com/saucelabs/saucebindings/GitManager.java

```diff
@@ -0,0 +1,24 @@
+package com.saucelabs.saucebindings;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class GitManager {
+    private static String currentBranch = "_default_";
```

Comment:
does such branch really exist?

Reply:
If you don't specify anything, Screener uses `(default)`, presumably to make sure it isn't a real branch. Unfortunately, this prevents me from clicking a link with that URL because my IDE because it doesn't like that character. So I changed the default to `_default_`, also unlikely to be the actual branch name, but I can click on resulting links.